frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/FeatureEnvironmentVariants.tsx

```diff
@@ -66,12 +66,8 @@ export const FeatureEnvironmentVariants = () => {
         patch: jsonpatch.Operation[];
         error?: string;
     } => {
-        try {
-            const updatedNewVariants = updateWeight(newVariants, 1000);
-            return { patch: createPatch(variants, updatedNewVariants) };
-        } catch (error: unknown) {
-            return { patch: [], error: formatUnknownError(error) };
-        }
+        const updatedNewVariants = updateWeight(newVariants, 1000);
+        return { patch: createPatch(variants, updatedNewVariants) };
```

Comment:
This allows the error to bubble back to the caller, which gives us a nice pretty error message. This should be prevented by the other changes for this error path but other cases can also cause failures, in which case an error message would be pretty cool